### PR TITLE
Fixing some questions at JSON-P docs

### PR DESCRIPTION
**Pull request by gabrielbussolo**
fixing #433 

~ps: I'm leaving it as a draft so the task progress can be tracked and errors and suggestions can be reviewed earlier~

### CODE REVIEW

File: install/jsonp/docs/index.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h2>Guide to Jakarta JSON Processing TCK 2.0 Documentation</h2>
 <ul>
   <li>The <em><a href="JSONP2.0-ReleaseNotes.html">Jakarta JSON Processing TCK Release Notes</a></em> provides the information on the test harness, the test suite and the platforms tested.
     <ul>
-      <li>If needed, updated  Release Notes are available at the Jakarta EE Specification web site for Jakarta JSON Processing: <a href="https://jakarta-ee/specifications/json-processing/2.0/">https://jakarta-ee/specifications/json-processing/2.0/</a></li>
+      <li>If needed, updated  Release Notes are available at the Jakarta EE Specification web site for Jakarta JSON Processing: <a href="https://jakarta.ee/specifications/jsonp/2.0/">https://jakarta.ee/specifications/jsonp/2.0/</a></li>
     </ul>
   </li>
   <li>The <em>Jakarta JSON Processing Technology Compatibility Kit, Version 2.0 User's Guide </em> provides the information that you need to install, set up, and  run  the Jakarta JSON Processing TCK, Version 2.0.  In addition, the guide provides the rules  you must comply with to pass the Jakarta JSON Processing TCK.</li>
```
Comment (alwin-joseph) [inline install/jsonp/docs/index.html:51]: Can you please also address this here as mentioned in https://github.com/eclipse-ee4j/jakartaee-tck/issues/433 
"Jakarta JSON Processing Technology Compatibility Kit, Version 2.0 User's Guide in the docs/index.html should be clickable link to actual user guide contained in the tck zip"

The link to the first page for html usersguide will be at html-usersguide/toc.html in the tck bundle (eg. https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-900/jakarta-persistence-tck-3.0.0.zip)

**Comment by gabrielbussolo** [comment]
@alwin-joseph do you know how can I get the assertion file for 2.0?

**Comment by alwin-joseph** [comment]
> @alwin-joseph do you know how can I get the assertion file for 2.0?

You can rename the assertion file for previous release, update the same with latest version, replace javax -> jakarta. Also the older assertion file for 1.0 could be deleted as well. for reference : https://github.com/eclipse-ee4j/jakartaee-tck/pull/439

This is all done manually now for this release. Ideally assertions are added when you have a new functionality, or when a functionality gets removed.

**Comment by gabrielbussolo** [comment]
> > @alwin-joseph do you know how can I get the assertion file for 2.0?
> 
> You can rename the assertion file for previous release, update the same with latest version, replace javax -> jakarta. Also the older assertion file for 1.0 could be deleted as well. for reference : #439
> 
> This is all done manually now for this release. Ideally assertions are added when you have a new functionality, or when a functionality gets removed.

@alwin-joseph  Thank you for your support, I believe that is ready to review. :smiley:

File: install/jsonp/docs/assertions/JSONP2.0JavadocAssertions.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
 <body bgcolor="white">
 <br>
 <CENTER>
-<h2>  Jakarta JSON Processing - 1.1<br>
+<h2>  Jakarta JSON Processing - 2.0<br>
```
Comment (RohitKumarJain) [inline install/jsonp/docs/assertions/JSONP2.0JavadocAssertions.html:27]: we need to update Copyright (c) 2018, 2019 to "Copyright (c) 2018, 2020"

Comment (gabrielbussolo) [reply install/jsonp/docs/assertions/JSONP2.0JavadocAssertions.html:27]: fixed :v:

Comment (gabrielbussolo) [reply install/jsonp/docs/assertions/JSONP2.0JavadocAssertions.html:27]: thanks

**Pull request merged**